### PR TITLE
Greg's feature flag C# post

### PR DESCRIPTION
**Pull request by Quorralyne**
This blog post has the following features:

- [x] A GitHub Repository with a polished README (N/A)
- [x] A GitHub Repository under the github.com/oktadev account (N/A)
- [x] A title that's approved by Dev Advocacy
- [x] A URL that's approved by Dev Advocacy
- [x] The content has been ran through Grammarly (https://www.grammarly.com/)
- [ ] Rendered locally and confirmed that no Markdown typos exist

### CODE REVIEW

File: _source/_posts/2021-07-28-toggle-feature-flags-csharp.md [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+
+Go to [ConfigCat.com](https://www.configcat.com) and sign up for a free account.
+
+You will see your first feature flag `My awesome feature flag` already created.
+
+{% img blog/toggle-feature-flags-csharp/image1.png alt:"" width:"800" %}{: .center-image }
+
+You can work with this feature flag, but I'm going to add a different one for my Twitter feed feature.
+
+{% img blog/toggle-feature-flags-csharp/image2.png alt:"" width:"800" %}{: .center-image }
```
Comment (bdemers) [inline _source/_posts/2021-07-28-toggle-feature-flags-csharp.md:117]: Alt tags would be great for SEO and accessiblity

File: _source/_posts/2021-07-28-toggle-feature-flags-csharp.md [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+## Takeaway
+
+Feature flags are quite simple to implement and add great value to your application's flexibility. Even if you choose to go with a feature flag service provider or implement your own solution, I'm confident your product team will love the idea of decoupling feature releases from code deployments, as it eliminates a major source of stress from the equation. Also, A/B testing capabilities and percentage-based roll-outs will save your brand reputation and increase confidence when going to production with a new idea.
+
+## Learn More About Feature Flags and Authentication
+
+- [Martin Fowler's blog post from 2017 on the topic, is a must read](https://martinfowler.com/articles/feature-toggles.html)
+- [Okta CLI, the simplest way to create secure application templates](https://developer.okta.com/blog/2020/12/10/introducing-okta-cli)
+- [Get Started with ASP.NET Core + Okta](https://developer.okta.com/blog/2020/12/15/okta-linux-dotnet-server-support)
+- [ConfigCat SDK for .NET manual and docs with advanced use cases](https://www.configcat.com/docs/sdk-reference/csharp)
```
Comment (bdemers) [inline _source/_posts/2021-07-28-toggle-feature-flags-csharp.md:222]: add social bits here

**Review by bdemers — APPROVED** [review]
a couple of nits

File: _source/_posts/2021-07-28-toggle-feature-flags-csharp.md (hunk 1 above)
Comment (bdemers) [inline _source/_posts/2021-07-28-toggle-feature-flags-csharp.md:117]:
```suggestion
{% img blog/toggle-feature-flags-csharp/image2.png alt:"Screen shot of foobar" width:"800" %}{: .center-image }
```

File: _source/_posts/2021-07-28-toggle-feature-flags-csharp.md (hunk 2 above)
Comment (bdemers) [inline _source/_posts/2021-07-28-toggle-feature-flags-csharp.md:222]:
```suggestion
- [ConfigCat SDK for .NET manual and docs with advanced use cases](https://www.configcat.com/docs/sdk-reference/csharp)

If you have any questions about this post, please add a comment below. For more awesome content, follow [@oktadev](https://twitter.com/oktadev) on Twitter, like us [on Facebook](https://www.facebook.com/oktadevelopers/), or subscribe to [our YouTube channel](https://www.youtube.com/oktadev).
```

File: _source/_posts/2021-07-28-toggle-feature-flags-csharp.md (hunk 1 above)
Comment (Quorralyne) [reply _source/_posts/2021-07-28-toggle-feature-flags-csharp.md:117]: I agree. I will work with contractors on providing that in the future.

File: _source/_posts/2021-07-28-toggle-feature-flags-csharp.md (hunk 2 above)
Comment (Quorralyne) [reply _source/_posts/2021-07-28-toggle-feature-flags-csharp.md:222]: Doh!

**Pull request merged**